src/main/kotlin/com/mparticle/kits/GoogleAnalyticsFirebaseGA4Kit.kt

```diff
@@ -62,14 +62,19 @@ class GoogleAnalyticsFirebaseGA4Kit : KitIntegration(), KitIntegration.EventList
         return listOf(ReportingMessage.fromEvent(this, mpEvent))
     }
 
-    override fun logScreen(s: String, map: Map<String, String>?): List<ReportingMessage> {
+    override fun logScreen(
+            screenName: String,
+            screenAttributes: Map<String, String>?
+    ): List<ReportingMessage> {
```

Comment:
```suggestion
        screenName: String,
        screenAttributes: Map<String, String>?
    ): List<ReportingMessage> {
```

The code looks good to me, just a formatting issue. You can fix it in Android Studio by going to:

Toolbar -> Code -> Reformat File